src/utils/wallet-seed.js

```diff
@@ -109,5 +112,12 @@ async function deriveEncryptionKey(password, salt, iterations, digest) {
 }
 
 export function lockWallet() {
-  setUnlockedMnemonicAndSeed(null, null);
+  setUnlockedMnemonicAndSeed(null, null, null);
+}
+
+// Returns the 32 byte key used to encrypt imported private keys.
+async function deriveImportsPrivateKey(mnemonic) {
+  const { mnemonicToSeed } = await import('bip39');
+  const rootSeed = Buffer.from(await mnemonicToSeed(mnemonic), 'hex');
+  return bip32.fromSeed(rootSeed).derivePath("m/501'/0'/0/0").privateKey;
```

Comment:
Note that the imported private keys are stored in local storage, encrypted by a key derived from the wallet's mnemonic.